activitysim/cli/run.py

```diff
@@ -298,14 +370,21 @@ def run(args):
     chunk.consolidate_logs()
     mem.consolidate_logs()
 
+    from ..core.flow import TimeLogger
+
+    TimeLogger.aggregate_summary(logger)
+
     tracing.print_elapsed_time("all models", t0)
 
+    if memory_sidecar_process:
+        memory_sidecar_process.stop()
+
     return 0
 
 
 if __name__ == "__main__":
 
-    from activitysim import abm  # register injectables
+    from activitysim import abm  # register injectables  # noqa: F401
 
     parser = argparse.ArgumentParser()
     add_run_args(parser)
```

Comment:
Sorry if this is outside of the scope of this PR, but is the `parser.parse_args(["--sum", "7", "-1", "42"])` below used? Seems like this line can be deleted.